app/models/post.rb

```diff
@@ -1,5 +1,14 @@
 class Post < ApplicationRecord
   belongs_to :user
```

Comment:
[REQUIRED]This should be `belongs_to :author, class_name: 'User'`. This helps differentiate between the different types of users